docs/agent/README.md

```diff
@@ -0,0 +1,36 @@
+# Datadog Agent
+
+## TOC
+
+ * [Changes][changes]
+ * [Configuration options][config]
+ * [Downgrade][downgrade]
+ * [GUI](gui.md)
+ * [Missing features][missing-features]
+ * [Upgrade][upgrade]
+
+## What is Agent 6?
+
+Agent 6 is the latest major version of the Datadog Agent. The big difference
+between Agent 5 and Agent 6 is that Agent 6 is a complete rewrite of the core
+Agent in Golang. Don’t worry! We still support Python checks.
+
+## Installation and management
+
```

Comment:
To me it'd make sense to add a reminder here that the agent 6 is currently in Beta, and link to the beta doc page